src/main/java/duke/FindCommand.java

```diff
@@ -0,0 +1,32 @@
+package duke;
+
+/**
+ * Encapsulate a command that allow users to find existing task with
+ * matching description, which is-a Command.
+ */
+public class FindCommand extends Command {
+
+    private String taskDescription;
```

Comment:
```suggestion
    private final String taskDescription;
```
Suggesting the addition of `final` modifier.

Reply:
I did not implement it as final as I may need to modify the said variable later on.